README.md

```diff
@@ -1,7 +1,5 @@
 # KOMOJU Magento Plugin
 
-- link to the support email for any concerns or questions
```

Comment:
haha, a good reminder why I should start all todos with `TODO:`, otherwise I won't spot it 😛